blackbox/docs/azure_discovery.txt

```diff
@@ -0,0 +1,49 @@
+.. highlight:: yaml
+.. _azure_discovery:
+
+===============
+Azure Discovery
+===============
+
+When running Crate on Microsoft Azure you usually face
+the problem that Crate's default discovery mechanism does
+not work out of the box.
+
+Luckily Crate has a built-in mechanism for unicast host
+discovery in an Azure environment.
+
+Requirements to use Azure api for node discovery
+------------------------------------------------
+
+The mechanisms for discovery on Microsoft Azure are built
+in with Crate, but the required sdk librarier are not included in
+any Crate distribution and needed to be added to Crate's Azure plugin
+libs folder. By default this is
+``$CRATE_HOME/plugins/cloud-azure/azure-libs``
+
+Please add the following libraries into the libs folder::
+
+    - azure-mgmt-utility-
+    ...
```

Comment:
don't you wanna list the libraries explicitly like we do here with HDFS? https://crate.io/docs/reference/sql/snapshot_restore.html#hadoop-2-x-required-libraries